tests/runtests.py

```diff
@@ -436,14 +455,20 @@ def test_cmd_pattern_create(self):
         self.assertIn("aaaaaaaabaaaaaaacaaaaaaadaaaaaaa", res)
         return
 
+    @include_for_architectures(valid_architectures=["x86_64", "aarch64"])
     def test_cmd_pattern_search(self):
-        cmd = "pattern search $rbp"
+        if ARCH == "aarch64":
+            r = "$x30"
+        if ARCH == "x86_64":
```

Comment:
elif

Reply:
not necessary

Reply:
It is if we are adding an exception case.